sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/CatalystConf.scala

```diff
@@ -29,6 +29,7 @@ trait CatalystConf {
   def groupByOrdinal: Boolean
 
   def optimizerMaxIterations: Int
+  def maxCaseBranches: Int
```

Comment:
Thank you for quick review. Sure. And also `maxCaseBranchesForCodegen` in SQLConf.scala.